components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/src/test/java/org.wso2.carbon.apimgt.rest.api.endpoint.registry/impl/RegistriesApiServiceImplTest.java

```diff
@@ -817,6 +856,57 @@ public void updateRegistryEntryWithDefinitionUrl() throws APIManagementException
         compareRegistryEntryDTOs(payloadEntryDTO, responseEntryDTO);
     }
 
+    @Test
+    public void updateRegistryEntryWithNoDefinitionFileAndUrl() throws APIManagementException {
+        final String REGISTRY_UUID = "reg1";
+
+        EndpointRegistryInfo endpointRegistryInfo = new EndpointRegistryInfo();
```

Comment:
shall we introduce a method to create a registryEntry when parameters are given? also another method to return a registryEntry with some default values.

Reply:
the reason is, if we are to add a new property to a schema there;s a lot of places to change to include them in the tests. By introducing a method we can minimize that effort.

Reply:
sure, will add that

Reply:
Added in https://github.com/wso2/carbon-apimgt/pull/8552/commits/e168f5df88c3d0fd01b64f4ad08d69c5bed0b45e